server/middlewares/checkauth.js

```diff
@@ -1,4 +1,4 @@
-const { CustomeError } = require('../utils/helper/customeError');
+const CustomeError = require('../utils/helper/customeError');
```

Comment:
require custome error from the index in utils 
like : const { CustomeError } = require('../../utils'); in (deleteMessage.js)